cfalertdialog/src/main/java/com/crowdfireapp/cfalertdialog/CFAlertDialog.java

```diff
@@ -647,4 +974,61 @@ public CFAlertDialog show() {
         private OnClickListener onSingleItemClickListener;
         private OnMultiChoiceClickListener onMultiChoiceClickListener;
     }
+
+    private static class CFAlertActionButton {
```

Comment:
Same as above. We are giving minimal flexibility in the default usage. For more flexibility we have support for custom UI. Again this can be picked up as future updates.